hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/SelfDestructSuite.java

```diff
@@ -18,6 +18,7 @@
 
 import static com.hedera.services.bdd.junit.TestTags.SMART_CONTRACT;
```

Comment:
My guess is that this suite needs to be in either `SequentialSuites` or `ConcurrentSuites`.  Probably `SequentialSuites`?